tests/testthat/test-gcalendr.R

```diff
@@ -1,64 +1,57 @@
-context("Read calendar events")
+test_that("Can read events", {
+  skip_if_no_token()
 
-if (Sys.getenv("GCALENDR_PASSWORD") != "" && gargle:::secret_can_decrypt("gcalendr")) {
-  test_that("Can read events", {
+  calendar_ids <- calendar_list()
+  expect_is(calendar_ids, "data.frame")
+  expect_equal(ncol(calendar_ids), 3)
 
-    secret <- gargle:::secret_read("gcalendr", "gmail-token")
-    token <- gargle::credentials_service_account(path = rawToChar(secret))
-    calendar_auth("apdevries@gmail.com", path = token, use_oob = TRUE)
 
-    expect_true(calendar_has_token())
+  # JENNY QUESTION: I'm worried about this. The test should expect to be
+  # logged in as the service account. But maybe I just don't understand what
+  # this function does.
```

Comment:
We should talk about this. You might need to auth as the service account and create some events, so we know what we expect to find here?

In googlesheets4, I'm putting the code that creates my test fixtures (Google Sheets in my case) here:

https://github.com/tidyverse/googlesheets4/tree/master/tools/test-fixtures

It's nice to have this programmatically recorded.